snmp/tests/common.py

```diff
@@ -284,6 +284,11 @@ def assert_common_metrics(aggregator, tags=None, is_e2e=False, loader=None):
 
 
 def assert_common_check_run_metrics(aggregator, tags=None, is_e2e=False, loader=None):
+    if is_e2e:
+        aggregator.assert_metric('snmp.device.reachable', metric_type=aggregator.GAUGE, at_least=0, value=1)
+        aggregator.assert_metric('snmp.device.unreachable', metric_type=aggregator.GAUGE, at_least=0, value=1)
+        aggregator.assert_metric('snmp.interface.status', metric_type=aggregator.GAUGE, at_least=0, value=1)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ah, maybe this is not needed since `assert_common_check_run_metrics` is only used in `loader:python` tests

Reply:
Also, the `aggregator.assert_metric` doesn't assert `tags`. Shall we add it?